src/app/components/Card.jsx

```diff
@@ -30,9 +36,20 @@ export default function CardSignUp() {
             priority
           />
         </div>
-      </div>
-      <div className={styles.container__slider_info}>
-        <h3>Speady, Easy and Fast</h3>
+        </div>
+      ) : (
+        <div className={styles.card_banner__sign_in}>
+         <Image
+          src="/revenue.svg"
+          alt="Ember logo"
+          width={370}
+          height={285}
+        />   
+        </div>
+      )}
+      <div className={isSignUpPage ? styles.container__slider_info : styles['container__slider_info--sign-in']}>
```

Comment:
Remember that when you use modifier classes, the original class must be present
```suggestion
      <div className={`${styles.container__slider_info}${isSignUpPage ? '' : ' ' + styles['container__slider_info--sign-in']}`}>
```